### DISCUSSION TITLE
FromDelimitedString

### DISCUSSION THREAD

**Discussion by Criperum** [davebronson/clear-hl7-net#11] [General]
Hi. Awesome lib, saved me tons of time and effort. As I see this feature is WIP right now. Are there any estimations on release? Or maybe branch is already stable?

**Comment by davebronson** [comment]
Hey @Criperum thanks for the compliment!  Glad that you're getting good use out of the library.  I wish that I could provide an accurate time estimate for completion of this enhancement, but the best I can do at this point is to say that I want to complete within the next 2-3 months.  I'll let you know when things are looking stable, and can put a beta out there.  Thanks!

**Comment by davebronson** [comment]
Hi @Criperum 

I have completed initial development on the `add-deserialization-methods` branch.  Units tests also created and running successfully.  I'm going to begin a phase of testing + tweaking here, and would like to get a beta version of the library out on NuGet within the next couple of weeks.  If you were looking to do some early testing, check out that branch and let me know if any issues.  And check issue #3 for further details.

Thanks!

**Reply by Criperum** [reply]
Hi @davebronson. I'll check new feature out ASAP. Thanks again for your effort.

**Comment by davebronson** [comment]
Hi @Criperum.  I've had the beta (with the new deserialization logic) on NuGet for a while without any negative feedback, and am also satisfied with my local testing.   I'm getting ready to merge and release, and wanted to see if you've had an opportunity to do any testing on your end?  No obligation, of course.  I appreciate any effort you've put towards helping double-check my handiwork.  Thanks!

**Reply by Criperum** [reply]
Hi @davebronson. I've made some testing and it works fine to me. Thanks for your efforts)))

**Reply by davebronson** [reply]
Excellent, thank you for the feedback.
